backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/input/restapi/mapper/UpdateModResponseMapper.java

```diff
@@ -0,0 +1,27 @@
+package de.bund.digitalservice.ris.norms.adapter.input.restapi.mapper;
+
+import de.bund.digitalservice.ris.norms.adapter.input.restapi.schema.UpdateModResponseSchema;
+import de.bund.digitalservice.ris.norms.application.port.input.UpdateModUseCase;
+import de.bund.digitalservice.ris.norms.domain.entity.Norm;
+import org.springframework.stereotype.Service;
+
+/** Mapper class for converting to a {@link UpdateModResponseSchema}. */
+@Service
+public class UpdateModResponseMapper {
+
+  // Private constructor to hide the implicit public one and prevent instantiation
+  private UpdateModResponseMapper() {}
+
+  /**
+   * Creates a {@link UpdateModResponseSchema} instance from the {@link Norm} entities.
+   *
+   * @param result The {@link UpdateModUseCase.Result} of the update use case.
+   * @return A new {@link UpdateModResponseSchema} instance.
+   */
+  public static UpdateModResponseSchema fromResult(final UpdateModUseCase.Result result) {
+    return UpdateModResponseSchema.builder()
+        .amendingNormXml(result.amendingNormXml())
+        .targetNormXml(result.targetNormXml())
```

Comment:
Why the target norm xml is needed? Is this the ZF0? is for the time machine? Oh wait we need to pass the ZF0 xml right? Then please rename to `zf0TargetNormXml` maybe?